### PR TITLE
Subscribe to vanish_requests stream for deletion

### PR DESCRIPTION
**Pull request by dcadenas**
This PR subscribes to the `vanish_requests` Redis stream and deletes data from the DB on each new request for a certain pubkey. I also updated the docker compose file to ease local testing with Redis and nosrelay, not just the firestore emulator.

I didn't add tests but I tested manually locally as it seems pretty safe and I'm worried about having time for the other tasks pending.

This is part of https://github.com/planetary-social/infrastructure/issues/139

### CODE REVIEW

**Comment by dcadenas** [comment]
I still need to update the ansible script to add the REDIS_URL

File: compose.yml
Comment (mplorentz) [inline compose.yml:1]: Is this the new preferred convention to just name this `compose.yml` instead of `docker-compose.yml`?

**Review by mplorentz — APPROVED** [review]
I just gave this a high level review, but it looks good! Great work!

File: compose.yml
Comment (dcadenas) [reply compose.yml:1]: Yes, I suspect that it's related to the change in which now it's `docker compose` vs `docker-compose` binary

**Pull request merged**